handlers/main.yml

```diff
@@ -1,3 +1,7 @@
 ---
 - name: Restart marathon
   service: name=marathon state=restarted
+
+- name: Reload daemon
+  command: systemctl daemon-reload
```

Comment:
Use systemd module: https://docs.ansible.com/ansible/systemd_module.html
Do do we need to support Ansible version <2.2?